modules/mono/editor/GodotTools/GodotTools.IdeMessaging/Client.cs

```diff
@@ -121,6 +121,7 @@ public Client(string identity, string godotProjectDir, IMessageHandler messageHa
             this.messageHandler = messageHandler;
             this.logger = logger;
 
+            // TODO: Need to fetch the project data dir name from ProjectSettings instead of defaulting to ".godot"
```

Comment:
@godotengine/mono What would be the proper way to fetch the project data directory name from the `ProjectSettings` and pass it here instead of the default value (`.godot`)?